src/utils/globalNavigasjonUtils.ts

```diff
@@ -124,6 +125,17 @@ function getNumberOfActiveSenOppfolgingOppgaver(
     : 0;
 }
 
+function getNumberOfFriskmeldingTilArbeidsformidlingOppgaver(
+  friskmeldingTilArbeidsformidlingVedtak: VedtakResponseDTO[]
+): number {
+  const sisteVedtak = friskmeldingTilArbeidsformidlingVedtak[0];
+  return !!sisteVedtak &&
+    !sisteVedtak.ferdigbehandletAt &&
+    !sisteVedtak.ferdigbehandletAt
+    ? 1
+    : 0;
```

Comment:
Dette holder ikke? 😊
```suggestion
  return sisteVedtak?.ferdigbehandletAt ? 0 : 1;
```

Reply:
Ser at jeg goofet litt på navne! Burde vært `sisteVedtak.ferdigbehandletBy`, altså både sjekke tidspunkt og veileder. Tenkte begge var greit å sjekke på, men skjønner jo at det ble litt mye ekstra når jeg sjekket samme felt to ganger 🙃

Hvis ingen vedtak er gjort før, blir `sisteVedtak === undefined` og det blir i så fall `false` og viser 1. Jeg har utvidet testene nå sånn at det sjekker alle disse casene også 👍🏼